src/app/components/submission-forms/actions.ts

```diff
@@ -127,6 +129,7 @@ export async function deleteSubmissionAction(
     await sql`
       DELETE FROM submissions
       WHERE submission_id = ${sqlSubmissionId}
+      AND author_id = ${session?.user?.providerAccountId!}
```

Comment:
oversight resolution. client side blocks exist but client anything can be spoofed, this check was missed earlier